### ISSUE TITLE
No indexing and "Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer"

### ISSUE THREAD

**Issue by JohnnyricoMC** [owncloud/search_lucene#17]
### Expected behaviour

Entering a string in the search bar returns filenames and files containing the provided string. Index files are created in /var/www/owncloud/data/username/lucene_index/
### Actual behaviour

In the file list only files containing the string in their filename change background like when users hover the cursor over the file's row. No search results appear. When Full Text Search is disables filenames get returned. No index files are created in /var/www/owncloud/data/username/lucene_index/, only a read.lock.file exists. No records exist in the oc_lucene_status table.
### Steps to reproduce

Created multiple user accounts, toggled Full Text search on and off. Entered strings appearing in stored text files into search box as well as strings matching filename.
### Server configuration:
- OS: Debian 7 "Wheezy"
- Web Server: Apache 2.2.22
- Database: MySQL 5.5.31
- PHP version: PHP 5.4.4-14
- Owncloud Version: 6.0.1
- Updated from an older owncloud or fresh install: yes, from Owncloud 5.0.14
- config/config.php:

``` <?php
<?php
$CONFIG = array (
  'instanceid' => '50eae36e2fdff',
  'passwordsalt' => 'OMITTED',
  'datadirectory' => '/var/www/owncloud/data',
  'dbtype' => 'mysql',
  'version' => '6.0.0.16',
  'dbname' => 'owncloud_mysql',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'owncloud_mysql',
  'dbpassword' => 'OMITTED',
  'installed' => true,

  'ldapIgnoreNamingRules' => false,
  'loglevel' => '0',
  'maintenance' => false,
  'theme' => '',
  'forcessl' => true,
  'maxZipInputSize' => 838860800,
  'allowZipDownload' => true,
);
```
- Are you using external storage, if yes which one: no
- All files and folders in /var/www/owncloud are currently owned by www-data
### Client configuration

Browsers: Mozilla Firefox, Google Chrome, Safari (Xubuntu and Mac OS X), untested on Windows
### Activated apps:
- Deleted files 0.4
- Encryption 0.5
- External storage support 0.2
- Full Text Search 0.5.2
- LDAP user and group backend 0.4.1
- PDF Viewer 0.2
- Share Files 0.3.5
- Text Editor 0.3
- Versions 1.3.3
#### Web server error log

IP addresses and domain name replaced.

```
[Fri Feb 14 15:32:31 2014] [error] [client 123.123.123.123] PHP Fatal error:  Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer in /usr/share/php/Zend/Search/Lucene/Analysis/Analyzer.php on line 71, referer: https://owncloud.domain.net/index.php/apps/files
[Fri Feb 14 15:32:31 2014] [error] [client 123.123.123.123] PHP Fatal error:  Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer in /usr/share/php/Zend/Search/Lucene/Analysis/Analyzer.php on line 71, referer: https://owncloud.domain.net/index.php/apps/files

```
#### Owncloud log

When setting cron method to AJAX, the entries below are triggered by entering a search term while Full Text Search is enabled. Essentially the same errors as in the web server error log.

```
{"app":"PHP","message":"Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer at \/usr\/share\/php\/Zend\/Search\/Lucene\/Analysis\/Analyzer.php#71","level":3,"time":"2014-02-14T14:26:41+00:00"}
{"app":"PHP","message":"Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer at \/usr\/share\/php\/Zend\/Search\/Lucene\/Analysis\/Analyzer.php#71","level":3,"time":"2014-02-14T14:26:42+00:00"}
{"app":"PHP","message":"Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer at \/usr\/share\/php\/Zend\/Search\/Lucene\/Analysis\/Analyzer.php#71","level":3,"time":"2014-02-14T14:29:44+00:00"}
{"app":"PHP","message":"Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer at \/usr\/share\/php\/Zend\/Search\/Lucene\/Analysis\/Analyzer.php#71","level":3,"time":"2014-02-14T14:29:44+00:00"}

```

Manual cron execution like I encountered in ticket #3 yields the following output:

```
$ sudo -u www-data /usr/bin/php -f /var/www/owncloud/cron.php 
PHP Fatal error:  Class 'Zend_Search_Lucene_Exception' not found in /var/www/owncloud/apps/search_lucene/document/Ods.php on line 35
Unexpected error!
```
### Possibly related to:

Issue #3 Class 'Zend_Search_Lucene_Document' not found

I'm at wit's end here, though that may be the end of the week talking. Any input, targeted questions, advice or remarks are appreciated.

Best regards,
Matthias

**Comment by equaeghe** [comment]
On 7.0.4 I am still hit by this bug:

```
Cannot redeclare class Zend_Search_Lucene_Analysis_Analyzer at /usr/share/php/Zend/Search/Lucene/Analysis/Analyzer.php#71
```

Note that I am on shared hosting, so I cannot touch `/usr`.

Searching has never worked for me. Sometimes, when I try it again, I get a message claiming indexing is going on, but nothing ever really seems to happen.

**Comment by karlitschek** [comment]
@butonic What do you think?

**Comment by butonic** [comment]
should be fixed with OC8, which contains search_lucene v0.6
